.gitignore

```diff
@@ -1,2 +1,3 @@
 legal
 start
+3.0/
```

Comment:
Ideally we'd put all results into a directory like `results`, and then ignore that.  For now, just ignore the most common target generated by `make`.